src/ui/VirtualFileRenderer/VirtualFileRenderer.tsx

```diff
@@ -0,0 +1,323 @@
+import { useWindowVirtualizer } from '@tanstack/react-virtual'
+// eslint-disable-next-line no-restricted-imports
+import type { Dictionary } from 'lodash'
+import Highlight, { defaultProps } from 'prism-react-renderer'
+import { useEffect, useLayoutEffect, useRef } from 'react'
+import { useHistory, useLocation } from 'react-router-dom'
+
+import { requestAnimationTimeout } from 'shared/utils/animationFrameUtils'
+import { cn } from 'shared/utils/cn'
+import { prismLanguageMapper } from 'shared/utils/prismLanguageMapper'
+
+import { ColorBar } from './ColorBar'
+
+// copied from prism-react-renderer since they don't export it
```

Comment:
Worth checking if you can infer this from anything they expose since it relates to a 3rd party dependency

Reply:
So I checked this out, and the version we're using there's really only this `const markup = Prism.languages['markup']` however the types don't actually align with the `Language` type and break things so there isn't really anything that we can use, at least with the version we're currently using.